crates/core/src/suppress.rs

```diff
@@ -39,21 +30,18 @@ impl<'a> Binding<'a> {
 }
 
 fn is_suppress_comment(
-    comment_node: &Node,
-    src: &str,
+    comment_node: &NodeWithSource,
     target_range: &Range,
     current_name: Option<&str>,
     lang: &impl Language,
 ) -> bool {
-    let child_range = comment_node.range();
-    let Ok(text) = comment_node.utf8_text(src.as_bytes()) else {
-        return false;
-    };
+    let child_range = comment_node.node.range();
+    let text = comment_node.text();
```

Comment:
`NodeWithSource::text()` has the same intention as `Node::utf8_text(src.as_bytes())`, but it doesn't return a `Result`. Theoretically it could panic, but that seems very unlikely because it means the node range is incorrect.